### ISSUE TITLE
Add locale support to date and time types

### ISSUE THREAD

**Issue by thiagodp** [thiagodp/concordialang#30]
### 1. New property `locale`

The property `locale` would assume the document's language by default.

Usage example:
```concordia
UI Element: Birth Date
  - data type is date
  - locale is "pt"
```
Every supported language should define the default formats for:
- date
- time
- datetime

### 2. New function `dtf`

> NOTE: I'm still not satisfied about using a function. Maybe special characters to denote a different kind of string could fit better.

The function means "*date time format*".

Currently there is the property `format`, which defines a UI Element's format using a *regular expression*. The proposed function aims to facilitate that definition by adopting a simpler format, like the [PHP date and time format](https://www.php.net/manual/en/function.date.php). 

Examples:
```concordia
UI Element: Birth Date
  - data type is date
  - format is dtf( "d/m/Y" )

UI Element: Meeting Hour
  - data type is time
  - format is dtf( "h:i" )

UI Element: Last Update
  - data type is datetime
  - format is dtf( "d/m/Y h:i:s" )
```

With the proposal, `format` should assume the locale's format by default. Example:
```concordia
UI Element: Birth Date
  - data type is date
# format is dtf( `m/d/Y` ) since the language is English and so is the locale.
```

**Comment by thiagodp** [comment]
See also #40

**Comment by thiagodp** [comment]
This feature is being released with [**Concordia 2**](https://github.com/thiagodp/concordialang/issues/56). The function `dtf` was replaced by a new UI Element property called `locale format` that uses [Unicode TR 35](http://www.unicode.org/reports/tr35/tr35-29.html#Date_Format_Patterns) as the date pattern. Example:
```concordia
UI Element: Start Date
  - locale format is "MM/dd"
```
The generated value will have only month and day.

**Comment by thiagodp** [comment]
Locale property released in 2.0-alpha. Function `dtf()` is not needed anymore, since language expressions were added.
